docs/DeveloperGuide.md

```diff
@@ -705,28 +703,26 @@ Priorities:
       
 *{More to be added}*
       
+## **Appendix D: Non-Functional Requirements**
 
-### Non-Functional Requirements
+1. Should work on any [`mainstream OS`](#common-classes) as long as Java `11` is installed and is the default.
+1. Should be able to hold up to 1000 deliverables, 1000 meetings and 1000 contacts without a noticeable sluggishness in performance for typical usage.
+1. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse. This excludes the Switch and Help command.
+1. The user interface should look intuitive and simple to navigate. It should not look cluttered with too many panels.
+1. The application should be for a single user, with its size being smaller than 100MB.
+1. The computer should have sufficient disk space to accommodate the user’s data stored in the system.
+1. The computer should have sufficient RAM and CPU power to run the JVM smoothly.
 
-1. Should work on any mainstream OS as long as it has Java 11 or above installed.
-1. Should be able to hold up to 1000 contacts without any negative effects.
-1. Should be able to hold up to 1000 deliverables without any negative effects.
-1. Should be able to schedule meetings up to 2 years in advance.
-1. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-*{More to be added}*
+## **Appendix E: Glossary**
 
-### Glossary
+* **Mainstream OS**: Windows, Unix, OS-X.
+* **OCP**: Overall Completion Percentage. It is a piechart showing the project's completion status, found on the left panel of the Dashboard.
+* **Mode**: an option allowing a change in the method of operation of the app. The app can be in dashboard, deliverable, meeting and contact mode.
```

Comment:
Sounds weird but idk what to change to

Reply:
Maybe change the first sentence to "The state of the application that affects how each command will be executed."